timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -38,6 +39,7 @@
     AutoETS=AutoETSModel,
     AutoARIMA=AutoARIMAModel,
     DynamicOptimizedTheta=DynamicOptimizedThetaModel,
+    RecursiveTabular=RecursiveTabularModel,
```

Comment:
which presets are we intended to add for this model?

Reply:
I plan to update these in a follow-up PR.